indico/modules/events/abstracts/blueprint.py

```diff
@@ -126,6 +126,8 @@
                  'email_tpl_text_edit', email_templates.RHEditEmailTemplateText, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/abstracts/email-templates/<email_tpl_id>',
                  'email_tpl_rest', email_templates.RHEmailTemplateREST, methods=('PATCH',))
+_bp.add_url_rule('/manage/abstracts/email-templates/quick_setup',
+                 'quick_setup_notifications', email_templates.RHQuickSetupEmailTemplates, methods=('GET',))
```

Comment:
I *think* that `GET` is the default, so you can just omit it
```suggestion
                 'quick_setup_notifications', email_templates.RHQuickSetupEmailTemplates)
```

Reply:
Acutally I'm not sure if `GET` is the best option here, `POST` or `PATCH` feels more appropriate

Reply:
Yes, a GET request should never perform modifications. Also, no underscores in URLs please, let's use `-` dashes instead

Reply:
I had the doubt since, even though it does modifications, it doesn't send any form and I was using an <a> tag for it, and <a> tags should send GET requests too